### DISCUSSION TITLE
Minimum utterance_end_ms

### DISCUSSION THREAD

**Discussion by fabriziobattiloro** [deepgram/community#563] [General help] [answered]
Good evening,
I'm facing a problem when i try to reduce the utterance_end_ms. When i choose an utterance fewer than 1000, this error appear: "Could not open socket: server rejected WebSocket connection: HTTP 400". Is there a way to reduce the utterance avoiding this problem?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by nikolawhallon** [comment, marked as answer]
The utterance end feature is based on word timings, and words can be detected in either final or interim results, since interim results are created around once every second, one can't specify utterance end values less than 1000 ms (for silence detection less than 1000 ms, endoointing is the way to go).

There's some more info here: https://developers.deepgram.com/docs/understanding-end-of-speech-detection#using-utteranceend

**Reply by fabriziobattiloro** [reply]
Thank you so much!
